core/common/src/main/java/alluxio/grpc/GrpcChannelKey.java

```diff
@@ -127,8 +87,6 @@ public boolean equals(Object o) {
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this)
-        .add("ClientType", mClientType)
```

Comment:
could you elaborate on why we want to remove client type and hostname here? I thought those are more for logging purpose